src/main/webapp/i18n/en/exam.json

```diff
@@ -586,6 +592,7 @@
                     "resolveComplaints": "Complaints can be resolved from the Assessment Dashboard.",
                     "testRun": "Try out different combinations of exercise variants from the student perspective.",
                     "exportResults": "Exports the exam results as a .csv file.",
+                    "editWorkingTime": "Until close to the end of the exam, you can extend its duration.",
```

Comment:
Can you mention the 5 minute limit here? "Close" is a bit vague.

Reply:
Sure, that was not 100% done as I was not quite happy with the naming anyways. And the deadline was also not fixed, might not be 5 minutes. Will leave this open till we have sth. fixed

Reply:
Mentioned the 5 minutes